### ISSUE TITLE
link problem in sys.version example

### ISSUE THREAD

**Issue by kubaroth** [dgrunwald/rust-cpython#80]
When building the first example (sys.version) from the Usage section I get the following link error, 
= note: /usr/bin/ld: cannot find -lpython3.4m
Do I need to specify more dependencies other then cpython = "0.1" ?
This is using rust-1.14, linux

**Comment by AraHaan** [comment]
What python version you have installed? you have python 3.4 or newer installed somewhere? it is possible if you do to install the python3.4m shared library there. I read something about it at some place. You might need to google for it sadly now.

**Comment by kubaroth** [comment]
I can run python3m  which points to 3.4.3
It feels like the search path is not looking into the right place.... Can the search path be extended from Cargo.toml or should I play with the LD_LIBRARY_PATH?
Also if possible I'd like to override version of python and target 2.7. How do I set this up?

**Comment by AraHaan** [comment]
yeah play with ``LD_LIBRARY_PATH`` and yes you can override it I think.

**Comment by wITTus** [comment]
For me, a symlink did the trick. E.g.:
```
root@ca29b4149d5d:/usr/lib/x86_64-linux-gnu# ln -s libpython3.5m.so.1 libpython3.5m.so
```
